composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
```

Comment:
You should leave this at `7.1`, so that it is as compatible as possible (as we want to install it in a wide range of applications).